pkg/cloudprovider/vsphereparavirtual/instances.go

```diff
@@ -44,7 +44,9 @@ type instances struct {
 }
 
 const (
-	providerPrefix = ProviderName + "://"
+	// providerPrefix is the Kubernetes cloud provider prefix for this
+	// cloud provider.
+	providerPrefix = "vsphere://"
```

Comment:
is it worth using a constant for `vsphere` to emphasize that `providerPrefix` should be in sync with whatever is used in `ProviderName()`?

Reply:
Yep